tests/agent_unittests/test_wrappers.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from newrelic.common.object_wrapper import function_wrapper
+
+
+@function_wrapper
```

Comment:
This should really be a fixture:
```suggestion
@pytest.fixture
def wrapper():
    @function_wrapper
    def wrapper(wrapped, instance, args, kwargs):
        return wrapped(*args, **kwargs)
        
    return wrapper
```

Reply:
After breaking the big test function apart I made these into fixtures anyway